trio/_tests/test_exports.py

```diff
@@ -301,7 +303,7 @@ def lookup_symbol(symbol):
             with mod_cache.open() as f:
                 return json.loads(f.read())["names"][name]
 
-    errors: dict[str, object] = {}
+    errors: Dict[str, object] = {}
```

Comment:
Why use `Dict` when we have the `from __future__ import annotations` block at the top?